processing/src/main/java/io/druid/guice/ExtensionsConfig.java

```diff
@@ -41,6 +41,10 @@
   @JsonProperty
   private String hadoopContainerDruidClasspath = null;
 
+  //Only applicable when hadoopContainerDruidClasspath is explicitly specified.
+  @JsonProperty
+  private boolean addExtensionsToHadoopContainer = false;
```

Comment:
minor nit: Maybe throw exception in case user has set this to true and hadoopContainerDruidClasspath is still null.